### DISCUSSION TITLE
More complete example with all form inputs like (select, radio, checkbox, radio group, checkbox group )

### DISCUSSION THREAD

**Discussion by Jarrodsz** [airjp73/rvf#198] [Q&A]
Could we have a example  with all the form inputs that are available?
Im learning remix and it goes beyond my skillset to figure out how to do radio and checkbox ( groups ) 

The custom components I am using from the example project:

FormInput.tsx
FormSelect.tsx
PasswordField.tsx
SubmitButton.tsx

Could the missing ones like radio and checkbox be added to repro with an example on how to validate them also?

**Comment by airjp73** [comment]
Hi!

Examples for these would be a great addition, but I probably don't have the time to add them. I'd be happy to accept a PR once you come up with something that works for you. 😊

It's not clearly labelled as an example, but you could use this test page as a reference.

* [Form with checkbox group](https://github.com/airjp73/remix-validated-form/blob/main/apps/test-app/app/routes/checkboxes.tsx)
* [Fieldset component in that example](https://github.com/airjp73/remix-validated-form/blob/main/apps/test-app/app/components/Fieldset.tsx)

For radio groups, make sure to validate it as a single value instead of an array like the checkbox group example.

**Reply by Jarrodsz** [reply]
ok im working on some of these, having a really hard time perhaps a list of common cases could be compiled and then an example can be made for each which would help a lot in adoption ( I guess this is only a problem for newbies ) probably.
